safe-global/safe-wallet-web
# [Multichain] fix: Check all fallbackHandler deployments [SW-236]

## What it solves

Resolves [SW-236](https://www.notion.so/safe-global/Unofficial-fallback-handler-message-for-cloned-safe-10e8180fe57380d4aa37fcd48feda821)

## How this PR fixes it

Uses `getCompatibilityFallbackHandlerDeployments` to get all deployment addresses in case there is a canonical and eip155 deployment and checks those addresses to decide whether its an official deployment or not

## How to test it

1. Open a Safe on a network with multiple deployments e.g. oeth:0xa4Fea8eF4a5dc35C5627Da7784b065934a5693a7
2. Go to the settings page and observe no unofficial fallback handler warning

## Screenshots

<img width="1265" alt="Screenshot 2024-09-30 at 11 32 13" src="https://github.com/user-attachments/assets/1d48259d-fb58-4510-be6b-1c83e75abcc9">


## Checklist
